src/django_components/expression.py

```diff
@@ -57,7 +62,90 @@ def resolve_string(
     return parser.compile_filter(s).resolve(context)
 
 
+def is_kwarg(key: str) -> bool:
+    return "=" in key
+
+
 def is_aggregate_key(key: str) -> bool:
     # NOTE: If we get a key that starts with `:`, like `:class`, we do not split it.
     # This syntax is used by Vue and AlpineJS.
     return ":" in key and not key.startswith(":")
+
+
+def process_aggregate_kwargs(kwargs: Mapping[str, Any]) -> Dict[str, Any]:
```

Comment:
Moved this here from `template_parser`, so `template_parser` contains only the template-parsing logic that we overrode from Django